### ISSUE TITLE
Layer version arn:aws:lambda:us-east-1:770693421928:layer:Klayers-python38-requests:5 does not exist.

### ISSUE THREAD

**Issue by routeronion** [jonrau1/SyntheticSun#26]
Jon,

I'm guessing this guy should be public but hitting the error that it doesn't exist when deploying.

Thanks,
David

**Comment by jonrau1** [comment]
Ah yeah that one is super old, latest is `arn:aws:lambda:us-east-1:770693421928:layer:Klayers-python38-requests:15`

**Comment by routeronion** [comment]
Thanks, I'll test and submit a PR if that is cool.

David

**Comment by jonrau1** [comment]
Yeah that would be great! I need to get back into this for a lot of refactors on the codebase too
